app/javascript/packs/application-common.js

```diff
@@ -16,6 +17,7 @@ import reactBlueprint from '../miq-component/react-blueprint';
 import * as helpers from '../miq-component/helpers';
 
 import { rxSubject, sendDataWithRx, listenToRx } from '../miq_observable';
+import { SnapshotForm } from '@manageiq/react-ui-components/dist/vm-snapshot-form';
```

Comment:
imho better to split this logic to another file, we should try keeping application-common to a minimum / top level

Reply:
Yep i agree this is in early phase i just needed to consult some things with @martinpovolny